src/main/java/uk/gov/hmcts/cmc/claimstore/utils/Formatting.java

```diff
@@ -34,13 +36,22 @@ public static String formatDate(LocalDateTime dateTime) {
         return formatTemporalWithPattern(dateTime, DATE_PATTERN);
     }
 
+    private static boolean decimalAmount(BigDecimal amount){
+        return amount.remainder(BigDecimal.ONE).compareTo(BigDecimal.ZERO) == 0;
+    }
+
     public static String formatDateTime(LocalDateTime dateTime) {
         return formatTemporalWithPattern(dateTime, DATE_TIME_PATTERN);
     }
 
     public static String formatMoney(BigDecimal amount) {
         requireNonNull(amount);
-        return NumberFormat.getCurrencyInstance(Locale.forLanguageTag("en-GB")).format(amount);
+        if(decimalAmount(amount)){
+            return decimalFormat.format(amount);
+        }
+        else{
+            return NumberFormat.getCurrencyInstance(Locale.forLanguageTag("en-GB")).format(amount);
```

Comment:
`NumberFormat.getCurrencyInstance(Locale.forLanguageTag("en-GB"))` can be a constant